web/src/components/organisms/Pagination.jsx

```diff
@@ -1,18 +1,16 @@
-// Pagination.jsx
 import { Pagination as FlowbitePagination } from 'flowbite-react'
-import { useState } from 'react'
 
-const Pagination = () => {
-  const [currentPage, setCurrentPage] = useState(1)
-
-  const onPageChange = (page) => setCurrentPage(page)
+const Pagination = ({ currentPage, totalPages, onPageChange }) => {
+  const handlePageChange = (page) => {
+    onPageChange(page)
+  }
 
   return (
     <div className='flex overflow-x-auto sm:justify-center'>
       <FlowbitePagination
         currentPage={currentPage}
-        totalPages={100}
-        onPageChange={onPageChange}
+        totalPages={totalPages}
+        onPageChange={handlePageChange}
```

Comment:
1. remove `handlePageChange`
then you can call`onPageChange` directly


```suggestion
        onPageChange={onPageChange}
```